api/resources_portal/test/dev_data/materials.json

```diff
@@ -0,0 +1,377 @@
+{
+  "materials": {
+    "0": {
+      "title": "Melanoma Reduction Plasmid",
+      "material_type": "plasmid",
+      "id": "1",
+      "url": "https://www.addgene.org/53246/",
+      "contact_user": "1",
+      "organization_id": "1",
+      "additional_metadata": {
+        "basic_info": {
+          "plasmid_name": "pGP1871",
+          "plasmid_type": "Encodes one insert",
+          "purpose": "Reduces melanoma growth rate",
+          "organism": "Danio rerio",
+          "Number of Availible Samples": "20",
+          "biosafety_level": "2"
+        },
+        "gene_insert": {
+          "gene_insert_name": {
+            "1": "Danio rerio",
+            "2": "Dani devrito"
+          },
+          "relevant_mutations": "Skin regrowth sequence modified",
+          "plasmid_backbone_name": "pSB1A30",
+          "primary_vector_type": "Bacterial Expression",
+          "cloning_method": "Unknown",
+          "sequence_maps": {
+            "1": "https://bucket-name.s3.region.amazonaws.com/keyname"
+          }
+        },
+        "growth_in_bacteria": {
+          "bacterial_resistance": "Kanamycin",
+          "copy_number": "High",
+          "growth_temp_celsius": "32.0",
+          "growth_strain": "Escherichia coli"
+        },
+        "additional_details": {
+          "contact_name": "Prim Proffer",
+          "contact_email": "pprof@harvard.edu",
+          "publication_title": "Expression Analysis of Zebrafish Melanoma",
+          "pre_print_doi": "10.1109/5.771073",
+          "pre_print_title": "Initial Research on Zebrafish Melanoma",
+          "citation": "Proffer, Prim. \"Expression Analysis of Zebrafish Melanoma\". March 15, 2020.",
+          "additional_info": "This paper was co-authored by Postworth Docktor."
+        }
+      },
+      "created_at": "2020-01-30 16:12:25-07",
+      "mta_s3_url": "https://bucket-name.s3.region.amazonaws.com/keyname",
+      "needs_irb": "false",
+      "needs_mta": "true",
+      "pubmed_id": "32223680",
+      "updated_at": "2020-01-30 16:12:25-07"
+    },
+    "1": {
+      "title": "Allele Extraction Protocol",
+      "material_type": "protocol",
+      "id": "2",
+      "url": "https://www.ncbi.nlm.nih.gov/pubmed/32223680",
+      "contact_user": "2",
+      "organization_id": "1",
+      "additional_metadata": {
+        "resource_details": {
+          "protocol_name": "Extraction of t-type alleles from zebrafish samples",
+          "description": "A protocol to retrieve t-type alleles relibably and without damage."
+        },
+        "additional_details": {
```

Comment:
Where'd the structure for additional_metadata come from? As in, having the keys `resource_details` and `additional_details`?

Reply:
It was based off of the sections of the UI. I'm not sure how many of the front-end routes have been made yet or how they're implemented- does this data need to be flattened?

Reply:
The routes will just return this as it is. Which fields are displayed where seems more like a frontend concern, but I'm not opposed if @dvenprasad, @davidsmejia, @arielsvn  would prefer us to split it in the data model.

Reply:
I agree with @kurtwheeler, we should flatten this and the frontend can represent it however we want.